packages/x-license/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license",
-  "version": "7.23.2",
+  "version": "7.23.5",
```

Comment:
Did Lerna suggest bumping this package? 🤔 
It doesn't look like it had any changes between `7.32.2` and now. 🤷

Reply:
I also skipped it for the one I did.

Reply:
Lerna suggested nothing 😄 
I had to use [--force-publish](https://lerna.js.org/docs/features/version-and-publish#synchronized-versions) to make it bump the packages.
I thought we usually bump `x-grid-generator` and `x-license` anyways, isn't it?

Reply:
Gotcha.
Did `--force-publish` suggest other product packages? 🤔 

> I'm ok with bumping it but why not go to 7.23.3?

Because we sync the package versions whenever we do a release. 😉

Reply:
I remember matching versions with the root version whenever we bumped a package, seems like my knowledge is a bit outdated in this regard. The recent releases do one version up, I'll match it here.

Btw, I'd love to have all these rules documented somewhere, seems like we do not stick to a specific strategy and keep changing time to time. 😕

Reply:
> Because we sync the package versions whenever we do a release. 😉

Do we? Damn, I just adjusted the versions to incrementing only one. 🤦

Reply:
> Did --force-publish suggest other product packages? 🤔

Yes, every package. I skipped the others manually.

Reply:
> Because we sync the package versions whenever we do a release. 😉

Let's have a consensus on it and make it part of `scripts/README.md`

@mui/x Are we OK on syncing the package versions whenever we release?

Reply:
PR opened: https://github.com/mui/mui-x/pull/16012

Reply:
OK, so we only miss the version update for `@mui/x-internals` and `@mui/x-license` and we should be good to go 👍🏻